components/metrics/BUILD.gn

```diff
@@ -20,8 +20,6 @@ static_library("metrics") {
     "data_use_tracker.h",
```

Comment:
please revert your formatting to this file and the other, it's third party code. They can be added to the list of excluded directories in .pre-commit-config.yaml if they haven't already

Reply:
yup, done. realized that as soon as i saw my cl on github. thanks